internal/pkg/containers/cri/cri_test.go

```diff
@@ -27,10 +27,10 @@ import (
 )
 
 const (
-	busyboxImage       = "docker.io/library/busybox:1.30.1"
-	busyboxImageDigest = "sha256:64f5d945efcc0f39ab11b3cd4ba403cc9fefe1fa3613123ca016cf3708e8cafb"
-	pauseImage         = "k8s.gcr.io/pause:3.1"
-	pauseImageDigest   = "sha256:da86e6ba6ca197bf6bc5e9d900febd906b133eaa4750e6bed647b0fbe50ed43e"
+	busyboxImage = "docker.io/library/busybox:1.30.1"
```

Comment:
I'm not sure how much it matters here, but i was thinking through this the other day. Would it be mo better to use scratch as our base image for comparison?  Not sure if that'd have any speed impact.

Reply:
Negligible. This downloads in seconds.